### DISCUSSION TITLE
archery是否有记录某个账号访问archery的所有域名变更记录

### DISCUSSION THREAD

**Discussion by JaySem** [hhyo/Archery#2307] [Q&A]
如题，archery的审计功能，是否有记录某1个登录账号，他在访问archery的过程中，记录所有访问archery的网址数据。比如访问某1时刻，访问查询页面的网址是A ，在另外1个时刻，访问另外1个页面的网址是B , archery内是否会把该账号访问A地址，和访问B地址的数据记录下来？

**Comment by JaySem** [comment]
如果有这个功能，就可以对每1个账号的访问域名，进行监控了

**Reply by LeoQuote** [reply]
一般此类功能由网关做比较合适
